README.md

```diff
@@ -112,6 +114,12 @@ such as restricting discovered features with the --label-whitelist option._
 | SSE4.2         | Streaming SIMD Extensions 4.2 (SSE4.2)
 | SGX            | Software Guard Extensions (SGX)
 
+### Memory Features
+
+| Feature name   | Description                                                                         |
+| :------------: | :---------------------------------------------------------------------------------: |
+| multinode      | Multiple nodes detected, sign of NUMA architecture
```

Comment:
Can we call this feature `numa`?

Reply:
The reason why I called it multinode is to keep the naming more neutral, i.e. NFD does not associate automatically "multiple nodes == numa", but instead express what NFD can read from system, i.e. "there is more than one node".
But I admit, this may in reality add confusion. In practice numa equals multinode everywhere, right, so lets leave separation to future when there may be multiple mappings. I will change the name to numa

Reply:
should it be NUMA or numa ?

Reply:
`numa` is fine.